mujoco_warp/testspeed.py

```diff
@@ -142,6 +160,8 @@ def _main(argv: Sequence[str]):
 
   with wp.ScopedDevice(_DEVICE.value):
     m = mjwarp.put_model(mjm)
+    if _EVENT_TRACE.value:
+      m.opt.graph_conditional = False  # graph conditional doesn't work with event trace
```

Comment:
nice, I like this.  thanks for thinking of it.

any time we manually override an option, let's print to stdout so the user knows we've done this.